src/MisraGries.cc

```diff
@@ -0,0 +1,44 @@
+//
+// Created by Nikolai Karpov on 2019-02-24.
+//
+#include "streamingcc_include/MisraGries.h"
```

Comment:
Hi Nikolai,
 can you keep the naming consistent (i.e., use lowercase to name the files). Thanks!